src/js/contentUtils.ts

```diff
@@ -316,7 +317,15 @@ export const scanForScripts = (): void => {
       mutationsList.forEach(mutation => {
         if (mutation.type === 'childList') {
           Array.from(mutation.addedNodes).forEach(checkScript => {
-            hasInvalidScripts(checkScript);
+            // Code within a script tag has changed
+            if (
+              checkScript.nodeType === 3 &&
+              mutation.target.nodeName.toLocaleLowerCase() === 'script'
+            ) {
+              hasInvalidScripts(mutation.target);
```

Comment:
Nit: not a new thing, but `hasInvalidScripts` is confusingly named. Seems like it would be marking something as being invalid rather than performing validation.